templates/blog/post_detail.html

```diff
@@ -0,0 +1,35 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>{{ post.title }}</h1>
+  <p>{{ post.content }}</p>
+  <small>{{ post.owner }}</small>
+  <small class="float-right">{{ post.created_time }}</small>
+  <hr>
+  <h2>Comments:</h2>
+  {% if post.commentaries.count %}
+    <ul class="list-group">
+      {% for comment in post.commentaries.all %}
+        <li class="list-group-item">
+          <small>{{ comment.user }}</small>
+          <small class="float-right">{{ comment.created_time }}</small>
+          <p>{{ comment.content }}</p>
+        </li>
+      {% endfor %}
+    </ul>
+  {% else %}
+    No comments.
+  {% endif %}
+  <hr id="leave-comment">
+  <h2>Leave a new comment:</h2>
+  <form action="{% url 'blog:commentary-create' post.pk %}" method="post" novalidate>
+    {% csrf_token %}
+    {% if user.is_authenticated %}
+      {{ form|crispy }}
+      <input class="btn btn-primary" type="submit" value="Submit"/>
```

Comment:
add cancel button as well

Reply:
There is not a single word about the cancel button in the readme. And why is it needed here?